packages/hooks/use-namespace/index.ts

```diff
@@ -51,7 +50,7 @@ export const useNamespace = (block: string) => {
       ? _bem(unref(namespace), block, blockSuffix, element, modifier)
       : ''
   const is = (name: string, state = true) =>
-    state ? `${statePrefix}${name}` : ''
+    name && state ? `${statePrefix}${name}` : ''
```

Comment:
I think the name here is required, we don't need this thing, besides, this method is internal use only, so our developer must know how it works.

Reply:
This feature can be used here. If `name` is a empty string, we just ignore it.

https://github.com/element-plus/element-plus/blob/6ff6270cdccd53507bda14830ccd0e07ce199c78/packages/components/select/src/select.vue#L92